src/OrbitGl/App.cpp

```diff
@@ -385,6 +386,13 @@ void OrbitApp::OnCaptureFinished(const CaptureFinished& capture_finished) {
         break;
     }
 
```

Comment:
In case `capture_finished.target_process_state_after_capture()` is `kCrashed`, what about adding a warning in the capture log? For example:
```
main_window_->AppendToCaptureLog(
    MainWindowInterface::CaptureLogSeverity::kWarning, GetCaptureTime(),
    absl::StrFormat("The target process crashed during the capture with signal %d.", capture_finished.target_process_termination_signal()));
```

Reply:
Yeah, makes sense I think.